triggers.gs

```diff
@@ -0,0 +1,77 @@
+/**
+* Trigger for thr rule for the number of executions for the script
+*/
+function mostExecutedTrigger() {
+  var files = DriveApp.getFilesByName("Admin Rules For Apps Scripts");
+  if (files.hasNext()) {
+    var file = SpreadsheetApp.open(files.next());
+    var active  = file.getSheetByName("Admin Rules"); 
+    var lastRow = active.getLastRow();
+    var lastCol = active.getLastColumn();
+    var values = active.getRange(lastRow, 1, 1, lastCol).getValues()[0];
+    var toTime = new Date();
+    var fromTime;
+    var millisInBetween;
+    var title;
+    if(values[1] == "EVERY_HOUR") { 
+      millisInBetween = 60*60*1000;
+      title = "Every hour";
+    } else if(values[1] == "EVERY_6_HOUR") {
+      millisInBetween = 6*60*60*1000;
+      title = "Every 6 hours";
+    } else if(values[1] == "EVERY_24_HOUR") {
+      millisInBetween = 24*60*60*1000;
+      title = "Every 24 hours";
+    } else if(values[1] == "EVERY_7_DAYS") {
+      millisInBetween = 7*24*60*60*1000;
+      title = "Every 7 days";
+    } else {
+      millisInBetween = 30*24*60*60*1000;
+      title = "Every 30 days";
+    }
+    
```

Comment:
Lines 13~30 can be extracted in a helper function

Reply:
Updated